SingularityUI/app/actions/api/requestGroups.es6

```diff
@@ -0,0 +1,6 @@
+import { buildApiAction } from './base';
+
+export const FetchGroups = buildApiAction(
+  'FETCH_GROUPS',
```

Comment:
i'd suggest being explicit with the naming (`FetchRequestGroups`, `FETCH_REQUEST_GROUPS`), we also have the concept of groups for authentication (which may be featured more prominently in the future)